src/timely-util/src/probe.rs

```diff
@@ -123,6 +130,12 @@ impl<T: Timestamp> Handle<T> {
             self.notify.notify_waiters();
         }
     }
+
+    #[inline]
+    fn reset_frontier(&mut self) {
+        self.frontier.borrow_mut().update_iter([(T::minimum(), 1)]);
+        self.handle_frontier = Antichain::from_elem(T::minimum());
+    }
```

Comment:
Ah, I should make this also retract the old `handle_frontier` from `frontier`. Changing this now!